src/gui/previewselect.h

```diff
@@ -63,11 +64,17 @@ class PreviewSelect: public QDialog, private Ui::preview
 protected slots:
     void on_previewButton_clicked();
     void on_cancelButton_clicked();
+    void on_preview_finished(int);
 
 private:
     QStandardItemModel *m_previewListModel;
     PreviewListDelegate *m_listDelegate;
     BitTorrent::TorrentHandle *const m_torrent;
+    bool headerState = false;
+    void restoreUi(Preferences *const pref);
+
+protected:
```

Comment:
move this section above `protected slots:`

Reply:
done